gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/SlackController.java

```diff
@@ -62,7 +64,9 @@ public SlackController(
     return slackChannelsCache.get();
   }
 
-  @Scheduled(fixedDelay = 300000L)
+  @Scheduled(
+      fixedDelayString = "${slack.channelRefreshIntervalInMs:1200000}",
+      initialDelayString = "${random.int(1200000)}")
```

Comment:
What's the reasoning behind randomizing this?

Reply:
all the instances poll at the same time, and we blow the rate limit